addon-test-support/@ember/test-helpers/setup-context.ts

```diff
@@ -161,6 +162,9 @@ export default function setupContext(
   let contextGuid = guidFor(context);
   CLEANUP[contextGuid] = [];
 
+  let testMetadata: ITestMetadata = getTestMetadata(context);
+  testMetadata.types.push('container');
```

Comment:
:sob: I don’t like this word for these type of tests :sad:

Reply:
Open to suggestions, but that's what the docs call them :)

https://guides.emberjs.com/release/testing/#toc_container-tests

Reply:
@rwjblue and I spoke offline. We decided that we'd rename `types` to `setupTypes` in order to capture the functions run during setup. We're also adding a `usedHelpers` to indicate whether rendering or visits had occurred.